<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObanalOptions</class>
 <widget class="QWidget" name="ObanalOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>658</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Редактор задания объективного анализа</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="typeGroupBox">
       <property name="title">
        <string>Тип данных</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="surfaceRadioButton">
          <property name="text">
           <string>Приземные</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="aeroRadioButton">
          <property name="text">
           <string>Аэрологические</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioRadioButton">
          <property name="text">
           <string>Радиолокационные</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="oceanRadioButton">
          <property name="text">
           <string>Морские</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="gribRadioButton">
          <property name="text">
           <string>&amp;GRIB</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="gribHourWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>130</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Срок прогноза:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="gribHourSpinBox">
            <property name="suffix">
             <string> час.</string>
            </property>
            <property name="maximum">
             <number>240</number>
            </property>
            <property name="singleStep">
             <number>3</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="sheduleGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Расписание</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QStackedWidget" name="sheduleStackedWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="basicPage">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Периодичность запуска:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="timesheetComboBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="advancedPage">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLineEdit" name="timesheetLineEdit">
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="placeholderText">
               <string>* * * * *</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="timesheetSetupButton">
              <property name="toolTip">
               <string>Открыть мастер настройки расписания</string>
              </property>
              <property name="text">
               <string>Настроить</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="advancedCheckBox">
            <property name="toolTip">
             <string>Настроить особое расписание запуска</string>
            </property>
            <property name="text">
             <string>Особое расписание</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="dateGroupBox">
     <property name="title">
      <string>Анализ данных за период</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QRadioButton" name="currentDateRadioButton">
        <property name="text">
         <string>Данные за текущую дату</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="customDateRadioButton">
        <property name="text">
         <string>Выбрать период</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="customDateTimeWidget" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QDateTimeEdit" name="startDateTimeEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="displayFormat">
            <string>dd.MM.yyyy hh:mm</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>−</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="endDateTimeEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="displayFormat">
            <string>dd.MM.yyyy hh:mm</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="levelsGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Анализируемые уровни</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLineEdit" name="levelsLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="levelClearButton">
        <property name="toolTip">
         <string>Установить значение по умолчанию</string>
        </property>
        <property name="text">
         <string>По умолчанию</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="levelSetupButton">
        <property name="toolTip">
         <string>Подробная настройка</string>
        </property>
        <property name="text">
         <string>Настроить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="descriptorsGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Анализируемые дескрипторы данных</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLineEdit" name="descriptorsLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="descriptorClearButton">
        <property name="toolTip">
         <string>Установить значение по умолчанию</string>
        </property>
        <property name="text">
         <string>По умолчанию</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="descriptorSetupButton">
        <property name="toolTip">
         <string>Подробная настройка</string>
        </property>
        <property name="text">
         <string>Настроить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="centersGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Центры обработки информации</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLineEdit" name="centersLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="centerClearButton">
        <property name="toolTip">
         <string>Установить значение по умолчанию</string>
        </property>
        <property name="text">
         <string>По умолчанию</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="centerSetupButton">
        <property name="toolTip">
         <string>Подробная настройка</string>
        </property>
        <property name="text">
         <string>Настроить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="stationsGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Метеорологические станции</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLineEdit" name="stationsLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stationClearButton">
        <property name="toolTip">
         <string>Установить значение по умолчанию</string>
        </property>
        <property name="text">
         <string>По умолчанию</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stationSetupButton">
        <property name="toolTip">
         <string>Подробная настройка</string>
        </property>
        <property name="text">
         <string>Настроить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="toolTip">
        <string>Сохранить изменения</string>
       </property>
       <property name="text">
        <string>Применить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="toolTip">
        <string>Закрыть без сохранения</string>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>applyButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>surfaceRadioButton</tabstop>
  <tabstop>aeroRadioButton</tabstop>
  <tabstop>radioRadioButton</tabstop>
  <tabstop>oceanRadioButton</tabstop>
  <tabstop>gribRadioButton</tabstop>
  <tabstop>timesheetComboBox</tabstop>
  <tabstop>advancedCheckBox</tabstop>
  <tabstop>currentDateRadioButton</tabstop>
  <tabstop>customDateRadioButton</tabstop>
  <tabstop>levelsLineEdit</tabstop>
  <tabstop>levelClearButton</tabstop>
  <tabstop>levelSetupButton</tabstop>
  <tabstop>descriptorsLineEdit</tabstop>
  <tabstop>descriptorClearButton</tabstop>
  <tabstop>descriptorSetupButton</tabstop>
  <tabstop>centersLineEdit</tabstop>
  <tabstop>centerClearButton</tabstop>
  <tabstop>centerSetupButton</tabstop>
  <tabstop>stationsLineEdit</tabstop>
  <tabstop>stationClearButton</tabstop>
  <tabstop>stationSetupButton</tabstop>
  <tabstop>startDateTimeEdit</tabstop>
  <tabstop>endDateTimeEdit</tabstop>
  <tabstop>gribHourSpinBox</tabstop>
  <tabstop>timesheetLineEdit</tabstop>
  <tabstop>timesheetSetupButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>oceanRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>levelsLineEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>oceanRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>levelClearButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>520</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>oceanRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>levelSetupButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>652</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
